bin/dump-to-git.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+var fs = require('fs');
+var process = require('process');
+var child_process = require('child_process');
+var Promise = require('bluebird');
+var NodeGit = require('nodegit');
+var LZString = require('lz-string');
+var models = require("./lib/models");
```

Comment:
oh sorry I missed it, that path is wrong (obviously).
it should be `../lib/models`.